extensions/banner/deployment/src/main/java/io/quarkus/banner/deployment/BannerConfig.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.banner.deployment;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "banner")
+public class BannerConfig {
+    /**
+     * The path of the banner which could be provided by user
```

Comment:
It should be mentioned mentioned that this path is a path relative to the root of the classpath, not a file system path